rules/windows/process_creation/win_exploit_cve_2017_0261.yml

```diff
@@ -20,8 +20,8 @@ logsource:
     product: windows
 detection:
     selection:
-        ParentImage: '*\WINWORD.EXE'
-        Image: '*\FLTLDR.exe*'
+        ParentImage|endswith: '\WINWORD.EXE'
+        Image|endswith: '\FLTLDR.exe*'
```

Comment:
it was `contains` in the original rule, better keep it if you are not sure that it should be `endswith`